source/Doffsetof.bs

```diff
@@ -0,0 +1,141 @@
+<pre class='metadata'>
+Title: Offsetof for Pointers to Members
+Shortname: Doffsetof
+Revision: 1
+Audience: SG1, EWG, CWG
```

Comment:
Audience should be EWG (the language Evolution Working Group). See: https://isocpp.org/std/the-committee